lib/jsonapi-serializers/serializer.rb

```diff
@@ -314,6 +312,10 @@ def self.serialize(objects, options = {})
       result
     end
 
+    def self.serialize_errors(errors)
+      { 'errors' => errors }
```

Comment:
For style consistency, please remove whitespace inside the hash. {'errors' => errors}